### PR TITLE
Update venv-redhat.rst

### PR DESCRIPTION
**Pull request by bkmgit**
Remove typelib-1_0-Pango-1_0 and tesseract-ocr as these are not in Fedora or Cent OS repositories
Replace redis-server with redis

Before submitting pull request, please ensure that:

- [ ] Every commit has message which describes it
- [ ] Every commit is needed on it's own, if you have just minor fixes to previous commits, you can squash them
- [ ] Any code changes come with testcase
- [ ] Any new functionality is covered by user documentation

### CODE REVIEW

File: docs/admin/install/venv-redhat.rst [hunk 1]
```diff
@@ -9,14 +9,14 @@ Installing on RedHat, Fedora and CentOS
...
-      cairo-devel typelib-1_0-Pango-1_0 gobject-introspection-devel libacl-devel \
+      cairo-devel gobject-introspection-devel libacl-devel \
       python3-pip python3-virtualenv python3-devel git
 
 .. include:: steps/install-system-optional.rst
 
 .. code-block:: sh
 
-    dnf install tesseract-ocr tesseract-devel leptonica-devel
+    dnf install tesseract-devel leptonica-devel
```
Comment (nijel) [inline docs/admin/install/venv-redhat.rst]: Language data are needed as well (that's what tesseract-ocr is supposed to pull), so probably this is better:

```suggestion
    dnf install tesseract-langpack-eng tesseract-devel leptonica-devel
```

File: docs/admin/install/venv-redhat.rst [hunk 2]
```diff
@@ -9,14 +9,14 @@ Installing on RedHat, Fedora and CentOS
 
    dnf install \
       libxslt-devel libxml2-devel freetype-devel libjpeg-devel zlib-devel libyaml-devel \
-      cairo-devel typelib-1_0-Pango-1_0 gobject-introspection-devel libacl-devel \
+      cairo-devel gobject-introspection-devel libacl-devel \
```
Comment (nijel) [inline docs/admin/install/venv-redhat.rst]: The devel package seems to include the gir data:

```suggestion
      cairo-devel pango-devel gobject-introspection-devel libacl-devel \
```

File: docs/admin/install/venv-redhat.rst (hunk 1 above)
Comment (bkmgit) [reply docs/admin/install/venv-redhat.rst]: Thanks.

**Comment by nijel** [comment]
Merged, thanks for your contribution!

**Pull request merged**